awsiot/iotidentity.py

```diff
@@ -339,18 +342,19 @@ def from_payload(cls, payload):
 class CreateKeysAndCertificateSubscriptionRequest(awsiot.ModeledClass):
     __slots__ = []
 
-    def __init__(self):
-        # type: () -> None
-        pass
+    def __init__(self, *args, **kwargs):
+        for key, val in zip([], args):
+            setattr(self, key, val)
 
 class ErrorResponse(awsiot.ModeledClass):
     __slots__ = ['error_code', 'error_message', 'status_code']
 
-    def __init__(self, error_code=None, error_message=None, status_code=None):
-        # type: (typing.Optional[str], typing.Optional[str], typing.Optional[int]) -> None
-        self.error_code = error_code # type: typing.Optional[str]
-        self.error_message = error_message # type: typing.Optional[str]
-        self.status_code = status_code # type: typing.Optional[int]
+    def __init__(self, *args, **kwargs):
+        self.error_code = kwargs.get('error_code')
+        self.error_message = kwargs.get('error_message')
+        self.status_code = kwargs.get('status_code')
+        for key, val in zip(['error_code', 'error_message', 'status_code'], args):
```

Comment:
Maybe add a comment like:
`# for backwards compatibility, read arguments that used to be accepted without keyword`